frontend/express/public/stylesheets/vue/clyvue.scss

```diff
@@ -23,6 +23,76 @@
     }
 }
 
+.cly-vue-notification {
+    &__alert-box {
+        align-items: center;
+        border: 1px solid;
+        box-sizing: border-box;
+        border-radius: 12px;
+        min-height: 48px;
+        font-weight: 400;
+        &--fullScreen {
+            width: 100%
+        }
+        &--inlineBlock {
+            display: inline-block;
+        }
```

Comment:
fullScreen can be just "full",
inlineBlock can be just "inline"

:slightly_smiling_face: